ext/configuration.h

```diff
@@ -184,6 +184,7 @@ enum ddtrace_sampling_rules_format {
     CONFIG(STRING, DD_VERSION, "", .ini_change = ddtrace_alter_dd_version,                                     \
            .env_config_fallback = ddtrace_conf_otel_resource_attributes_version)                               \
     CONFIG(STRING, DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP, DD_TRACE_OBFUSCATION_QUERY_STRING_REGEXP_DEFAULT) \
+    CONFIG(BOOL, DD_TRACE_MEMCACHED_OBFUSCATION, "true")                                                       \
```

Comment:
Yes: Ruby and Python's behavior before those envs were implemented was to not include the command at all. The env includes the command, rather than unobfuscating it.